amundsen_application/static/js/components/SearchPage/SearchPanel/tests/index.spec.tsx

```diff
@@ -1,95 +1,34 @@
 import * as React from 'react';
 
 import { shallow } from 'enzyme';
-import { SearchPanel, SearchPanelProps, mapStateToProps} from '../';
-import globalState from 'fixtures/globalState';
-import { ResourceType } from 'interfaces/Resources';
-import { TABLE_RESOURCE_TITLE, USER_RESOURCE_TITLE } from 'components/SearchPage/constants';
+import SearchPanel, { SearchPanelProps } from '../';
 
-import AppConfig from 'config/config';
+import ResourceSelector from 'components/SearchPage/ResourceSelector';
 
 describe('SearchPanel', () => {
   const setup = (propOverrides?: Partial<SearchPanelProps>) => {
     const props = {
-      selectedTab: ResourceType.table,
-      tables: globalState.search.tables,
-      users: globalState.search.users,
-      dashboards: globalState.search.dashboards,
-      onTabChange: jest.fn(),
+      onChange: jest.fn(),
       ...propOverrides
     };
     const wrapper = shallow<SearchPanel>(<SearchPanel {...props} />);
     return { props, wrapper };
   };
 
-  describe('onChange', () => {
-    // TODO - placeholder as I expect the `onChange` to disappear once we use redux for search state actions
-  });
-
-  describe('renderRadioOption', () => {
-    // TODO - placeholder as I expect the radio buttons logic to change
-  });
 
   describe('render', () => {
     let props;
     let wrapper;
-    let renderRadioOptionSpy;
 
     beforeAll(() => {
       const setupResult = setup();
       props = setupResult.props;
       wrapper = setupResult.wrapper;
-      renderRadioOptionSpy = jest.spyOn(wrapper.instance(), 'renderRadioOption');
-    });
-
-    it('renders the table resource option', () => {
-      renderRadioOptionSpy.mockClear();
-      wrapper.instance().render();
-      expect(renderRadioOptionSpy).toHaveBeenCalledWith(ResourceType.table, TABLE_RESOURCE_TITLE, props.tables.total_results);
     });
 
-    it('renders the user resource option when enabled', () => {
-      AppConfig.indexUsers.enabled = true;
-      renderRadioOptionSpy.mockClear();
-      wrapper.instance().render();
-      expect(renderRadioOptionSpy).toHaveBeenCalledWith(ResourceType.user, USER_RESOURCE_TITLE, props.users.total_results);
-    });
-
-    it('does not render user resource option when disabled', () => {
-      AppConfig.indexUsers.enabled = false;
-      renderRadioOptionSpy.mockClear();
-      wrapper.instance().render();
-      expect(renderRadioOptionSpy).not.toHaveBeenCalledWith(ResourceType.user, USER_RESOURCE_TITLE, props.users.total_results);
+    it('renders a ResourceSelector', () => {
+      expect(wrapper.find(ResourceSelector).exists()).toBe(true)
```

Comment:
nit: Can check the props for verifying not just that `SearchPanel` renders its children, but also passes in the correct things to it's children.

Reply:
^ Unless that is temp logic.

Reply:
Yeah, the `onChange` prop is only here until we can use redux actions for changing selected resources.